src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -68,6 +68,32 @@ object Import {
     )
   }
 
+  case class Bytes(underlying: Long) extends AnyVal {
+    def round1k: Bytes =
+      Bytes((Math.max(underlying - 1, 0) >> 10 << 10) + 1024)
+  }
+
+  object ByteConversions {
+    implicit class LongToBytes(value: Long) {
```

Comment:
Using a `Long` here will open the door for overflows for the below methods. Maybe use an `Int`?